Dockerfile

```diff
@@ -29,5 +29,5 @@ ADD --chown=app_user gunicorn_logging.conf .
 
 # You can set the number of workers by passing --workers=${NB_WORKER} to the docker run command.
 # For some reason, an array is required here to accept other params on run.
-ENTRYPOINT ["gunicorn", "app:app", "--bind=0.0.0.0:5000", "--pid=pid", \
-  "--worker-class=meinheld.gmeinheld.MeinheldWorker", "--preload", "--log-config=/app/gunicorn_logging.conf"]
+ENTRYPOINT ["uvicorn", "app:app", "--host=0.0.0.0", "--port=5000", \
+  "--log-config=/app/gunicorn_logging.conf"]
```

Comment:
In a production environment, uvicorn documentation recommends to keep gunicorn, using `UvicornWorker` as the worker class. https://www.uvicorn.org/deployment/#gunicorn

This also makes possible to keep our current deployment and logging configurations, to use multiple workers by using `--workers` in the docker command, and to keep the `--preload` option to reduce the memory usage.